rover/src/owr_drive_controls/include/crabSteer.hpp

```diff
@@ -0,0 +1,24 @@
+/*
+ * Defines the data structure used to do crab steering
```

Comment:
I please include license  and other stuff specified in our standa

Reply:
Done